src/app/shared/model/audio-version.model.ts

```diff
@@ -106,7 +109,7 @@ export class AudioVersionModel extends BaseModel implements HasUpload {
         this.explicit = 'Clean';
         break;
       default:
-        this.explicit = '';
+        this.explicit = undefined;
```

Comment:
I thought we didn't use undefined or null on the model fields? I got tripped up on that 9+ months ago with explicit and have never really understood what the issue was with using it, something about messing up change detection?